docs/Growing.md

```diff
@@ -117,6 +117,132 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
+
+An important difference between mojo.js and other web frameworks is that it can operate in two modes, both as a 
+full-fledged web framework in single-file mode, and as a micro web framework optimized for rapid prototyping.
+
+### Differences
+
+You likely know the feeling, you've got a really cool idea and want to try it as quickly as possible. That's exactly
+why mojo.js apps can be run as a single file (plus package.json for deps).
+
+```
+package.json // Will be generated when you install mojo
+myapp.js     // templates can be inlined in the file.
+```
+
+Full mojo.js applications on the other hand are much closer to a well organized NPM distribution to maximize
+maintainability.
+
+```
+my_app                                // Application dir created manually
+|- config.yml                         // Configuration file
+|- index.js                           // Application script
+|- node_modules/
+|   |- *lots of node files*
+|- package.json                       // Node package information and settings
+|- package-lock.json                  // Describes exact tree generated in node_modules/
+|- test/                              // Test directory
+|   |- example.js                     // Random test
+|- controllers/                       // Controller directory
+|   |- example.js                     // Controller containing route actions
+|- models/                            // Model directory
+|- public/                            // Static file directory (served automatically)
+|   |- index.html                     // Static HTML file
+|- views/                             // Views directory
+|   |- example/                       // View directory for "Example"
+|   |   |- welcome.html.ejs           // EJS template generating HTML
+|   |- layouts/                       // View directory for layouts
+|   |   |- default.html.ejs           // Layout view/template
+```
+
+These application skeletons can be automatically generated with the mojo commands `generate-lite-app` and
+`generate-full-app` - in both cases you need to create a folder for the skeleton and run `npm i @mojojs/core`
+first to install mojolicious for your new app.
+
+```
+$ mkdir myapp && cd myapp
+$ npm i @mojojs/core
+$ npx mojo generate-full-app myapp # or
+$ npx mojo generate-lite-app myapp
+```
+
+Featurewise both are almost equal, the only real differences are organizational, so each one can be gradually
```

Comment:
"Featurewise" is a typo.

Reply:
Should have known better than trusting Wiktionary. 🤦🏻 Will revert to the original spelling.